tests/baselines/reference/classDeclarationCheckUsedBeforeDefinitionInItself.js

```diff
@@ -4,6 +4,9 @@ class C3 {
 }
 
 //// [classDeclarationCheckUsedBeforeDefinitionInItself.js]
-class C3 {
-}
-C3.intance = new C3(); // ok
+var C3 = (() => {

```

Comment:
This should use `let` instead of `var` to have the same scope and TDZ as the class declaration that was there before

Reply:
Indeed. It should.  It was on my cleanup list and now implemented.